.npmignore

```diff
@@ -0,0 +1,13 @@
+# Warning!
+# This file must be the same as .gitignore, but MUST NOT CONTAIN "dist"
+
+node_modules/
+npm-debug.log
+.DS_Store
+*.iml
+.idea/
+/.project
+reports
+/website/storybook
+/website/component-docs/doc*
+/website-build
```

Comment:
According to the docs:
> Like git, npm looks for .npmignore and .gitignore files in all subdirectories of your package, not only the root directory.
Besides, this commit just changes where we are actually releasing from, which won't hit a parent `.npmignore`, as far as I understand it. Good question, though -- had to look it up :)